schedule/alp/alp_yast.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: alp_yast
+description: >
+    Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+    Enable automation for YaST
```

Comment:
change to 'Enable YaST automation test for ALP'?

Reply:
Done.